File: Java/Car.java
public class Car {
    private int id;
    private String license;
    private Account driver;
    private int passenger;

    public Car(String license, Account driver) { // esto es el metodo constructor
        this.license = license;
        this.driver = driver; //

    }

    void printDataCar() {
        if (passenger != 0) {
            System.out.println("License: " + license + "Driver:" + driver.name + "Passengers:" + passenger);

        }
    }

    public int getPassenger() {
        return passenger;
    }

    public void setPassenger(int passenger) {
        if (passenger == 4) {

            this.passenger = passenger;
        } else {
            System.out.println("necesitamos asignar 4 pasajeros");
        }

    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

}

File: PHP/Car.php
<?php
require_once("Account.php");
class Car
{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function _construct($license, $driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar()
    {
        echo ("license: $this->license
        conductor:   {$this->driver->name}  
        Numero de pasajeros:   $this->passenger");
    }

    public function getPassenger()
    {
        return $this->passenger;
    }

    public function setPassenger($passenger)
    {
        if ($passenger == 4) {
            $this->passenger = $passenger;
        } else {
            echo "Ocupas asignar 4 pasajeros";
        }
    }
}
